travis/build.sh

```diff
@@ -0,0 +1,13 @@
+#!/bin/bash
+set -ex
+if [[ "$TRAVIS_PULL_REQUEST" != "false" ]]; then     # pull request
+    bundle exec fastlane test
+elif [[ -z "$TRAVIS_TAG" && "$TRAVIS_BRANCH" == "mPower-Android" ]]; then  # non-tag commits to master branch
+    bundle exec fastlane test
+    bundle exec fastlane assemble
+elif [[ -z "$TRAVIS_TAG" && "$TRAVIS_BRANCH" =~ ^mPower-Android-.* ]]; then # non-tag commits to stable branches
```

Comment:
The tagging happens in fastlane/Fastfile in the 'alpha' lane (or function).  it will be tagged with the same convention currently used for ios apps, $verNum_$buildNum, so it will look something like 1.0_2, 1.0_3, etc..  Basically it will look exactly like tags for elevateMS repo.